.travis.yml

```diff
@@ -56,12 +72,15 @@ before_script:
     fi
   - |
     if [[ "$WP_TRAVISCI" == "lint" ]] ; then
+      set -e
       composer global require wp-coding-standards/wpcs
       phpcs --config-set installed_paths $HOME/.composer/vendor/wp-coding-standards/wpcs
       npm run lint:js
+      set +e
     fi
 
 script:
+  - set -e
```

Comment:
This [means that](https://ss64.com/bash/set.html) if a command exits with non-zero, the build will fail. 

For example, if `npm run env start` fails, the e2e test will fail also. So it makes sense to let it display the error, and stop the build.